src/js/Controllers/UIController.js

```diff
@@ -91,16 +92,21 @@ class UIController {
                     // Generic HMI only supports main window for now.
                     return false;
                 }
+                var showApp = store.getState().appList.find((app) => {
+                    return app.appID === rpc.params.appID;
+                });
+                var showAppIsMedia = showApp.appType.includes('MEDIA');
```

Comment:
HMIApplicationStruct has an additional (redundant) parameter to identify an app type. `isMediaApplication`
```suggestion
                var showAppIsMedia = showApp.appType.includes('MEDIA') || showApp.isMediaApplication;
```

untested ^

Reply:
This param is only in the `OnAppRegistered` notification and the app list state is only updated by the `UpdateAppList` notification, the `isMediaApplication ` value is not stored anywhere in the state.

I thought this value was redundant as well, do you think it is worth adding this to the state? It wouldn't be straight forward to do because the OnAppRegistered arrives before UpdateAppList.

Reply:
I am seeing isMediaApplication sent in the UpdateAppList rpc
<img width="566" alt="Screen Shot 2021-08-12 at 2 21 40 PM" src="https://user-images.githubusercontent.com/12038416/129248634-8691eee4-6a8d-4798-82fa-cd92f0bafed3.png">

Reply:
Ah I was mistaken about that. That is the proper value to check, I have changed the appType checks in https://github.com/smartdevicelink/generic_hmi/pull/419/commits/0bcbcf1456effdc9fc18467f7f8134beac13abe6